docs/conf.py

```diff
@@ -186,8 +186,8 @@
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-    (master_doc, 'reana.tex', 'reana Documentation',
-     'info@reana.io', 'manual'),
+    # (master_doc, 'reana.tex', 'reana Documentation',
+    #  'info@reana.io', 'manual'),
```

Comment:
Is this necessary? If we remove PDF and DPUB targets on RTFD, isn't it enough?  Asking because it seems easier to disable it there in the settings.... and if somebody wants to build a PDF version locally, they would still be able to...